wiki/blocks/first-block.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Adding your first Block
+index: 1
+---
+# Adding your first Block
+Creating a Block is pretty similar to creating an Item, [read the corresponding article](../items/first-item) for more elaborate explanations.
+
+## Creating and registering the Block
+
+First we create the Block and store it in a field:
+
+```java
+public final Block EXAMPLE_BLOCK = new Block(new QuiltBlockSettings())
+```
+
+Then we register it in the `onInitialize()` function:
+
+```java
+Registry.register(Registries.BLOCK, new Identifier(mod.metadata().id(), "example_block"), EXAMPLE_BLOCK);
+```
+## Adding a BlockItem
+
+Having done that, we can place the block using the `setblock` command, but there isn't an item to place it. So we additionially register a `BlockItem` for the block and add it to the `BUILDING_BLOCKS` item group:
+
+```java
+Registry.register(Registries.ITEM, new Identifier(mod.metadata.id(), "example_block"), new BlockItem(EXAMPLE_BLOCK, new QuiltItemSettings()))
+
+ItemGroupEvents.modifyEntriesEvent(ItemGroups.BUILDING_BLOCKS).register(entries -> {
+	entries.addStack(EXAMPLE_BLOCK.asItem().getDefaultStack());
+});
+```
+
+For more information on what this does, see the [Creating your first Item](../items/first-item#registering-the-item) Article.
+
+## Adding a Model for the Block
+
+First, we need to create the BlockStates file for your Block. This tells the game which BlockStates use which models. For an example on using a more complicated BlockState setup, see [Adding redstone functionality to your Block](TODO). In this case, we only have one BlockState and can use this simple json:
+```json
```

Comment:
```suggestion
`assets/simple_block_mod/blockstates/example_block.json`:
```json
```
Moved path from after code block and added file name.
Also added `assets/simple_block_mod/` because I think it's important to make the path as clear as possible. Asset paths are a common issue, especially for new modders.